### PR TITLE
Preventing a container's filesystem from filling up (e.g. log files)

### PR DESCRIPTION
**Pull request by baude**
Added a section on what developers can do to help prevent
container filesystems from filling up.

### CODE REVIEW

**Comment by baude** [comment]
@imcleod @eliskasl can you guys review this?

**Comment by eliskasl** [comment]
LGTM

File: planning/planning_index.adoc [hunk 1]
```diff
@@ -301,9 +303,47 @@ Custodia is an open-source project that defines an API for storing and sharing s
...
+
 ==== User NameSpace Mapping (docker-1.10 feature)
 ==== https://www.openshift.com/promotions/docker-security.html
 
+=== Preventing your image users from filling the filesystem
+
+Most default docker deployments only set aside about 20GB of storage for each container.  For many applications, that
+amount of storage is more than enough.  But, if your containerized application performs a lot of logging or rarely would
+be recycled, filesystem space can be a concern.
+
```
Comment (imcleod) [inline planning/planning_index.adoc:316]: Suggested edit: "But if your containerized application produces significant log output, or your deployment scenario restarts containers infrequently, file system space can become a concern."

File: planning/planning_index.adoc [hunk 2]
```diff
@@ -213,11 +214,11 @@ several solutions for logging containers like:
 As a developer, if your application does using logging of some manner, you should be thinking about how you will
```
Comment (scollier) [inline planning/planning_index.adoc]: Typo, "using" -> "uses"

**Pull request merged**